app/views/content/non-uk-teachers/find-a-teaching-job-as-a-non-uk-citizen.md

```diff
@@ -0,0 +1,125 @@
+
+
+### Salaries and benefits 
+
+Schools have some flexibility on [teachers’ salaries](/is-teaching-right-for-me/teacher-pay-and-benefits). It varies by region and your level of skill, experience and qualifications. 
+
+There are [specific salary requirements for a Skilled Worker visa](/non-uk-teachers/visas-for-non-uk-teachers/#salary-requirement-for-a-skilled-worker-visa).
+
+
+### Get help applying for teaching jobs 
+
+As part of your application, schools will usually ask for your: 
+
+* educational and professional qualifications 
+
+* employment history 
+
+* personal statement 
+
+* references 
+
+Read up on how to: 
+
+* [write a great teaching job application in 5 steps](https://teaching-vacancies.campaign.gov.uk/get-help-applying-for-teacher-jobs/write-a-great-teaching-job-application) 
+
+* [write a teacher personal statement](https://teaching-vacancies.service.gov.uk/jobseeker-guides/how-to-write-teacher-personal-statement)  
+
+* [prepare for a teaching interview lesson](https://teaching-vacancies.campaign.gov.uk/get-help-applying-for-teacher-jobs/prepare-for-a-teaching-interview-lesson)  
+
+* [approach a teaching job interview](https://teaching-vacancies.campaign.gov.uk/get-help-applying-for-teacher-jobs/how-to-approach-a-teaching-job-interview) 
+
+
+## Apply for your visa 
+
+You’ll need a job offer from a school before you apply for your Skilled Worker visa.   
+
+You can apply for a Skilled Worker visa up to 3 months before your new job starts.
+
+[Apply for your visa to teach in England](/non-uk-teachers/visas-for-non-uk-teachers) has more information. 
+
+ 
+
+## Career progression 
+
+In your first teaching job after you qualify, you’ll complete an early career teacher (ECT) 2-year induction period, when you’ll work and be assessed as a paid teacher in a school. 
+
+This induction period is part of the UK government’s [early career framework](https://www.gov.uk/government/publications/early-career-framework) for teachers in England, which helps teachers succeed by developing their professional practice, knowledge and working habits. You’ll get training and support from a mentor to help you meet the [Teachers’ standards](https://www.gov.uk/government/publications/teachers-standards). 
+
+You’ll also be able to develop your career through the [national professional qualifications framework](https://www.gov.uk/guidance/national-professional-qualifications-framework). 
+
+Learn more about [moving up the career ladder in teaching](/is-teaching-right-for-me/career-progression).  
+
+## Financial incentives 
+
+You may be eligible for the:
+
+* [levelling up premium payment](https://www.gov.uk/guidance/levelling-up-premium-payments-for-teachers)
+* [early career payment](https://www.gov.uk/guidance/early-career-payments-guidance-for-teachers-and-schools)
+
+## Contact
+
+If you have general questions about the information on this page, you can email us at teach.inengland@education.gov.uk.
+
```

Comment:
@laurahermionetennant92 Seems to be a lot of additional whitespace at the end of this file that needs deleting?